helm/templates/client/deployment.yaml

```
@@ -0,0 +1,131 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "cpnConsole.fullname" . }}-client
+  labels:
+    {{- include "cpnConsole.client.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.client.autoscaling.enabled }}
+  replicas: {{ .Values.client.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "cpnConsole.client.selectorLabels" . | nindent 6 }}
+  strategy:
+    type: {{ .Values.client.strategy.type }}
+  template:
+    metadata:
+      annotations:
+        {{- include "checksum" (list $ "/client/configmap.yaml") | nindent 8 }}
+        {{- include "checksum" (list $ "/client/secret.yaml") | nindent 8 }}
+        {{- with .Values.client.podAnnotations }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+      labels:
+        {{- include "cpnConsole.client.selectorLabels" . | nindent 8 }}
+        {{- with .Values.client.podLabels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- if and .Values.imageCredentials.username .Values.imageCredentials.password }}
+      imagePullSecrets:
+      - name: {{ include "cpnConsole.name" . }}-pullsecret
+      {{- end }}
+      securityContext:
+        {{- toYaml .Values.client.podSecurityContext | nindent 8 }}
+      {{- if .Values.client.initContainers }}
+      initContainers:
+        {{- toYaml .Values.client.initContainers | nindent 8 }}
+      {{- end }}
+      containers:
+      - name: {{ .Chart.Name }}
```

```suggestion
      - name: client
```
Mieux vaut un nom fixe ici, plus facile pour écrire des scripts reproductibles et pas de risque de conflit (tant que le nom du Pod est différent)